GOVERNANCE.md

```diff
@@ -0,0 +1,94 @@
+
+Currently SIG membership requires invitation - potential qualified members are Active Contributors in a particular module. It is our intention to properly guide or mentor community talents, and help them advance to the higher-level Reviewer, Committer and Maintainer within an SIG.
+
+Each SIG must have a charter that specifies its scope (topics, code repos and directories), responsibilities, areas of authority, how members and roles of authority/leadership are selected/granted, how decisions are made, and how conflicts are resolved. See the SIG charter process for details on how charters are managed. SIGs should be relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-SIG processes.
+
+See [sig governance](/sig-governance/sig-governance.md) for more details about current SIG operating mechanics, such as mailing lists, meeting times, etc.
+
+#### Working Groups
+
+A Working Group is formed by community developers who gather together to accomplish a specific goal. To achieve the goal, some WGs may span over multiple SIGs, and some WGs may only focus on something specific in a specific SIG.
+
+Each WG has its life cycle. Once the goal is completed, the group will be disbanded. The only goal of WG operations and management is to ensure that the goals set by the group are completed at the right time. In general, the working group hold periodic meetings to summarize the current project progress and determine the plans for next steps.
+
+<!---to add Working Group governance link -->
+
+## Contributions
+
+Anyone can contribute to the project, regardless of their skills, as there are many ways to contribute. The various ways of contributing are described in more detail in a separate document. For details, see [Contributing to TiKV](https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md).
+
+## Support
+
+All participants in the community are encouraged to provide support for new users within the project management infrastructure. This support is provided as a way of growing the community. Those seeking support should recognize that all support activity within the project is voluntary and is therefore provided as and when time allows. A user requiring guaranteed response times or results should therefore seek to purchase a support contract from a community member. However, for those willing to engage with the project on its own terms, and willing to help support other users, the community support channels are ideal.
+
+## Decision Making Process
+
+Decisions about the future of the project are made by the PMC. New proposals and ideas (changes of major features, organization, or processes) can be brought to the PMC’s attention through the [Request for Change (RFC)](https://github.com/tikv/rfcs)process. For the change to happen, the RFC must earn the majority votes in the PMC.
```

Comment:
So although maintainers "set technical direction and priorities for the sub-project" and "make or approve design decisions for the project", but a features proposal still needs majority votes to get it merged?

Reply:
Agree.

Reply:
Maybe need further clarification here.  What about now?